package.json

```diff
@@ -110,6 +110,9 @@
     "helmet": "^3.14.0",
     "isomorphic-fetch": "^2.2.1",
     "js-cookie": "^2.2.0",
+    "moment": "^2.24.0",
```

Comment:
We shouldnt need `moment` as its inc in `moment-timezone` :)

Reply:
👍 thanks